src/main/java/nextstep/courses/domain/session/SessionImage.java

```diff
@@ -1,11 +1,29 @@
 package nextstep.courses.domain.session;
 
-import nextstep.file.ImageFile;
+import nextstep.file.domain.ImageFile;
 
 public class SessionImage {
 	ImageFile imageFile;
 
-	public SessionImage(int size, int width, int height, String type) {
+	public SessionImage(Long id) {
+		this.imageFile = new ImageFile(id);
+	}
+
+	public SessionImage(ImageFile imageFile) {
+		valid(imageFile.getSize(), imageFile.getWidth(), imageFile.getHeight(), imageFile.getTypeString());
+		this.imageFile = imageFile;
+	}
+
+	public SessionImage(Long id, int size, int width, int height, String type) {
+		valid(size, width, height, type);
+		this.imageFile = new ImageFile(id, size, width, height, type);
```

Comment:
```suggestion
		this(new ImageFile(id, size, width, height, type));
```
부생성자가 주생성자를 호출해 구현할 것을 추천
이렇게 구현할 경우 유효성 체크와 같은 중복 코드를 제거하는 효과가 생긴다.